app/core/functions.php

```diff
@@ -0,0 +1,77 @@
+<?php require 'database.php'; 
+
+/*
+*
+*
+* Global Functions 
+*
+*
+*/
+
+/* ###### INIT ###### */
+
+function init() {
+    // Default Page
+    $page = "search";
+    // get URL
+    $url = parseURL();
+    // update page if set
+    if(isset($url[0])){
+        if(file_exists('controllers/'. $url[0] . '.php')){
+            $page = $url[0];
+        }
+        else {
+            $page = '404';
+        }
+    }
+
+    // Require Controller
+    require_once 'controllers/' .$page. '.php';
```

Comment:
To expand on my other comments.  Here I'm thinking when we have the "user" page which shows a list of all their reports, we're pretty much going to be using the same "search.php" function, except it will have a `user_id` param or something similar.  So maybe the controller name shouldn't be tightly coupled to the view?